src/flamegraph.js

```diff
@@ -606,18 +646,6 @@ export default function () {
     return chart
   }
 
-  chart.transitionDuration = function (_) {
-    if (!arguments.length) { return transitionDuration }
-    transitionDuration = _
-    return chart
-  }
-
-  chart.transitionEase = function (_) {
```

Comment:
Some users depend on these functions. I'm Ok with not having it as a default, but option should be there.

Reply:
Well, problem is that now transitions are CSS based and I don't see how to keep these functions "functional", e.i. I can see how we can keep them, but they will not work.

Reply:
We have a great D3 guy here in the office, so I'll ping him see if he has any ideas.